ragna/deploy/_ui/app.py

```diff
@@ -23,17 +22,13 @@
 
 
 class App(param.Parameterized):
-    def __init__(self, *, hostname, port, api_url, origins, open_browser):
```

Comment:
Does `open_browser` have a new home? I noticed that `ragna deploy` doesn't take me to my browser anymore.

Reply:
Yeah that is kinda unfortunate, but no way around it for now. `uvicorn` has no option to open the browser on startup. I have some ideas on how to get around that and will open an issue. For now, let's not block the overall development over this.

Reply:
See #430.